src/main/java/org/myrobotlab/framework/repo/IvyWrapper.java

```diff
@@ -453,14 +470,14 @@ synchronized public void install(String location, String[] serviceTypes) {
       FileIO.toFile("libraries/install.sh", sb.toString().getBytes());
 
       Ivy ivy = Ivy.newInstance(); // <-- for future 2.5.x release
-      ivy.getLoggerEngine().pushLogger(new IvyWrapperLogger(Message.MSG_INFO));
+      ivy.getLoggerEngine().pushLogger(new IvyWrapperLogger(msgLevel));
 
       ResolveReport report = null;
       List<String> err = new ArrayList<>();
       try {
-         report = Main.run(cmd);
-      } catch(Exception e) {
-         err.add(e.toString());
+        report = Main.run(cmd.toArray(new String[cmd.size()]));

```

Comment:
Don't need `cmd.size()` here, the only reason a String array is needed is to work around generic type erasure. So you can do `new String[0]`, reduces clutter

Reply:
heh .... I find the String[0] cluttery as well ;)